keras_core/export/export_lib.py

```diff
@@ -81,32 +83,65 @@ def __init__(self):
         self._endpoint_signatures = {}
         self.tensorflow_version = tf.__version__
 
-        self._tf_trackable = tf.__internal__.tracking.AutoTrackable()
-        self._tf_trackable.variables = []
-        self._tf_trackable.trainable_variables = []
-        self._tf_trackable.non_trainable_variables = []
+        if backend.backend() == "tensorflow":
+            self._tf_trackable = tf.__internal__.tracking.AutoTrackable()
+            self._tf_trackable.variables = []
+            self._tf_trackable.trainable_variables = []
+            self._tf_trackable.non_trainable_variables = []
+        else:  # JAX backend
+            self.module = tf.Module()
```

Comment:
Use a private attribute -- also is this the right class? Why is AutoTrackable inappropriate for JAX?

Reply:
Unified with AutoTrackable, thanks for the great suggestion!